prime-angular-frontend/src/app/shared/enums/college-licence-grouping.enum.ts

```diff
@@ -4,14 +4,52 @@ export enum CollegeLicenceGroupingEnum {
   RPN = 3,
   NP = 4,
   MW = 5,
+  CertifiedDentalAssistant = 6,
+  DentalHygienist = 7,
+  DentalTechnicain = 8,
+  DentalTerapist = 9,
+  Dentist = 10,
+  Denturist = 11,
+  HealthProfessionOfDietetics = 12,
+  HealthProfessionOfOccupationalTherapy = 13,
+  HealthProfessionOfOpticianry = 14,
+  HealthProfessionOfOptomety = 15,
+  HealthProfessionOfPhysicalTherapy = 16,
+  HealthProfessionOfPsychology = 17,
+  HealthProfessionOfAudiology = 18,
+  HealthProfessionOfHearingInstrumentDispensing = 19,
+  HealthProfessionOfSpeechLanguagePathology = 20,
+  HealthProfessionOfChiropractic = 21,
+  HealthProfessionOfMassageTherapy = 22,
+  HealthProfessionOfNaturopathicMedicine = 23,
+  HealthProfessionOfTraditionalChineseMedicineAcupuncture = 24
```

Comment:
Is this intended to be aligned with `CollegeLicenseGroupingConfiguration.cs`?  E.g. Dentist value differs 🤔

Reply:
What do you mean?
![image](https://github.com/user-attachments/assets/fc754e5f-3925-4374-b1b0-e984ee778ea5)

Reply:
Sorry, I was probably looking at `Weight` incorrectly.  That said, a couple of typos in `DentalTechnicain` and `DentalTerapist`

Reply:
Ops. it is fixed.